trusted-firmware-m/secure_fw/spm/cmsis_func/main.c

```diff
@@ -150,6 +151,10 @@ int main(void)
 
     fih_delay_init();
 
+    if (!hw_unique_key_are_any_written()) {
```

Comment:
Does this need to be invoked before tfm_core_init(), which is supposed to do a lot of core initialization stuff?